src/stealth/stealth.js

```diff
@@ -36,6 +36,7 @@ const DEFAULT_STEALTH_OPTIONS = {
 };
 
 const STEALTH_ERROR_MESSAGE_PREFIX = 'StealthError';
+const STEALTH_COUNTER_MESSAGE = 'Stealth evaluated!';
```

Comment:
Maybe instead of having this as a constant, we could randomly generate one hash and console.log that hash? It would be a bit harder to detect.